pkg/compose/compose.go

```diff
@@ -92,3 +92,34 @@ func escapeDollarSign(marshal []byte) []byte {
 	escDollar := []byte{'$', '$'}
 	return bytes.ReplaceAll(marshal, dollar, escDollar)
 }
+
+// projectFromName builds a types.Project based on actual resources with compose labels set
+func (s *composeService) projectFromName(containers Containers, projectName string) *types.Project {
```

Comment:
This is basically a duplicate for `projectFromLabels`, better use/adapt this existing function

Reply:
Yes, that's right but `projectFromName` exists in the file `down.go ` and using that didn't seem right structure-wise.
I would handle this by removing it from `down.go` and modifying the code to use this one instead.
Does it sound good?

Reply:
yes, move it to `compose.go`. Also, probably should be renamed `projectFromActualResources` or something comparable to be more explicit

Reply:
- Removed `projectFromLabels` from `down.go`
- Fixed an issue causing some tests to fail
I couldn't think of a short and non-misleading alternative for `projectFromName` tho.